### ISSUE TITLE
mathjax如何配置

### ISSUE THREAD

**Issue by jiangchengyike** [AlynxZhou/hexo-theme-aria#3]
我按照README讲的设置了CDN之后，发现不仅没有成功，页面也没法正常显示了。

**Comment by AlynxZhou** [comment]
请问你是怎么配置的？麻烦你把自己的config贴上来，然后打开网页的开发者控制台（F12或者Ctrl+Shift+i），检查console下有没有报错，发给我。

**Comment by jiangchengyike** [comment]
mathjax: true 
  cdn: https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML
  global:
# Note: `fontawesome` is fontawesome-all. `scrollspy` and `util` are from bootstrap. `busuanzi` is `busuanzi.pure.mini.js`.
# If you don't know what you are doing, don't change it.
lib_cdn:
  enable: true
  jquery:
  fontawesome:
  scrollspy:
  util:
  lightgallery:
    js:
    css:
  lg_fullscreen:
  lg_hash:
  lg_pager:
  lg_thumbnail:
  lg_zoom:
  busuanzi:
  atom-one-dark:
  atom-one-light:
  solarized-dark:
  solarized-light:
  normalize:
  search:
  marked:
  timeago:
  commentjs:
    js:
    css:
主要就是改了这些，但是在运行hexo g命令时会报错
YAMLException: bad indentation of a mapping entry at line 67, column 6:
      cdn: https://cdn.mathjax.org/mathja ... 
         ^
    at generateError (/home/kiff/web/node_modules/js-yaml/lib/js-yaml/loader.js:165:10)
    at throwError (/home/kiff/web/node_modules/js-yaml/lib/js-yaml/loader.js:171:9)
    at readBlockMapping (/home/kiff/web/node_modules/js-yaml/lib/js-yaml/loader.js:1080:7)
    at composeNode (/home/kiff/web/node_modules/js-yaml/lib/js-yaml/loader.js:1332:12)
    at readDocument (/home/kiff/web/node_modules/js-yaml/lib/js-yaml/loader.js:1492:3)
    at loadDocuments (/home/kiff/web/node_modules/js-yaml/lib/js-yaml/loader.js:1548:5)
    at Object.load (/home/kiff/web/node_modules/js-yaml/lib/js-yaml/loader.js:1569:19)
    at Hexo.yamlHelper (/home/kiff/web/node_modules/hexo/lib/plugins/renderer/yaml.js:7:15)
    at Hexo.tryCatcher (/home/kiff/web/node_modules/bluebird/js/release/util.js:16:23)
    at Hexo.<anonymous> (/home/kiff/web/node_modules/bluebird/js/release/method.js:15:34)
    at Promise.then.text (/home/kiff/web/node_modules/hexo/lib/hexo/render.js:61:21)
    at tryCatcher (/home/kiff/web/node_modules/bluebird/js/release/util.js:16:23)
    at Promise._settlePromiseFromHandler (/home/kiff/web/node_modules/bluebird/js/release/promise.js:512:31)
    at Promise._settlePromise (/home/kiff/web/node_modules/bluebird/js/release/promise.js:569:18)
    at Promise._settlePromise0 (/home/kiff/web/node_modules/bluebird/js/release/promise.js:614:10)
    at Promise._settlePromises (/home/kiff/web/node_modules/bluebird/js/release/promise.js:693:18)
    at Async._drainQueue (/home/kiff/web/node_modules/bluebird/js/release/async.js:133:16)
    at Async._drainQueues (/home/kiff/web/node_modules/bluebird/js/release/async.js:143:10)
    at Immediate.Async.drainQueues [as _onImmediate] (/home/kiff/web/node_modules/bluebird/js/release/async.js:17:14)
    at runCallback (timers.js:696:18)
    at tryOnImmediate (timers.js:667:5)
    at processImmediate (timers.js:649:5)

**Comment by AlynxZhou** [comment]
注意这个报错 `YAMLException: bad indentation of a mapping entry at line 67, column 6:` 是你的yaml语法不对，这里说你缩进错了，请注意原来config的缩进…

**Comment by jiangchengyike** [comment]
改了下，虽然没有错误，但是还是没办法正常显示数学表达式
